external/java/src/main/java/ch/epfl/dedis/lib/crypto/bn256/GFp2.java

```diff
@@ -179,8 +179,8 @@ GFp2 square(GFp2 a) {
     }
 
     GFp2 invert(GFp2 a) {
-        BigInteger t = a.y.multiply(a.y);
-        BigInteger t2 = a.x.multiply(a.x);
+        BigInteger t = a.y.multiply(a.y).mod(p);
```

Comment:
This doesn't make a difference because everything is in mod p. We use these values later so keeping them small is a minor performance improvement (but I didn't do a benchmark)

Reply:
Changing it back because there's no evidence that this helps.